client/data/marketplace/use-es-query.ts

```diff
@@ -95,39 +102,45 @@ const getWpLocaleBySlug = ( slug: LanguageSlug ) => {
 	return languages.find( ( l ) => l.langSlug === slug )?.wpLocale || defaultLanguage;
 };
 
+export const getFetchESPluginsInfinite = (
```

Comment:
Needed to extract the `react-query` cache key and the fetch function pair in order to use them in the server prefetch middleware.